### DISCUSSION TITLE
TRS-80 Color Computer support?

### DISCUSSION THREAD

**Discussion by mrgw454** [spotlessmind1975/ugbasic#539] [General]
Looks like a great project.  Since there is support for the Dragon 32/64 (6809) any thought of adding support for Tandy Color Computer (at least the CoCo 1 and 2) since they both use the 6847 VDP?  Tandy Color Computer 3 with extended support for it's additional video modes (via the GIME) would be nice as well.  Either way, I'll be following this project.  Thanks!

**Comment by spotlessmind1975** [comment]
Hello @mrgw454 , TRS-80 CoCo computer support has been added (target: `coco`).

Please refer to this page:
https://github.com/spotlessmind1975/ugbasic/issues/124
